core/network/src/main/java/com/teamwable/network/util/createImagePart.kt

```diff
@@ -0,0 +1,17 @@
+package com.teamwable.network.util
+
+import android.content.ContentResolver
+import android.net.Uri
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import okhttp3.MultipartBody
+
+suspend fun ContentResolver.createImagePart(uriString: String?): MultipartBody.Part? {
+    if (uriString.isNullOrEmpty()) return null
+    val uri = runCatching { Uri.parse(uriString) }.getOrElse { return null }
```

Comment:
```suggestion
    val uri = runCatching { Uri.parse(uriString) }.getOrNull()
```
p2 : getOrNull()도 있습니당

Reply:
사실 uri parse하는데 오류는 없겠지만.. 만약 오류가 생길시에 즉시 return하려고 위의 코드를 사용 했어요!
아래 코드는 함수는 계속 실행되고 최종적으로 null이 할당된다 하네요!

Reply:
아하이하아하!!! I got it goooood